listenbrainz/webserver/static/js/src/RecentListens2.test.tsx

```diff
@@ -0,0 +1,248 @@
+import { enableFetchMocks } from "jest-fetch-mock";
```

Comment:
RecentListens2 is not a descriptive name. Maybe `RecentListensControls.test.tsx`? Why did we add a second file though? Could we not put these tests in a Seperate describe block in the old file?

Reply:
done